grpc-server/src/main/scala/com/wavesplatform/api/grpc/TransactionsApiGrpcImpl.scala

```diff
@@ -81,10 +82,8 @@ class TransactionsApiGrpcImpl(wallet: Wallet,
 
   override def broadcast(tx: PBSignedTransaction): Future[PBSignedTransaction] = {
     commonApi
-      .broadcastTransaction(tx.toVanilla)
-      .map(_.toPB)
-      .resultE
-      .toFuture
+      .broadcastTransaction(tx.toVanilla, forceBroadcast = true)
```

Comment:
Shouldn't this behave just like HTTP API, which respects `settings.allowTxRebroadcasting`?